core/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalStatsTests.java

```diff
@@ -32,13 +32,12 @@
     @Override
     protected InternalStats createTestInstance(String name, List<PipelineAggregator> pipelineAggregators,
                                                Map<String, Object> metaData) {
-        long count = randomIntBetween(1, 50);
-        double[] minMax = new double[2];
-        minMax[0] = randomDouble();
-        minMax[0] = randomDouble();
-        double sum = randomDoubleBetween(0, 100, true);
-        return new InternalStats(name, count, sum, minMax[0], minMax[1], DocValueFormat.RAW,
-            pipelineAggregators, Collections.emptyMap());
+        long count = randomIntBetween(0, 50);
```

Comment:
Might be worth using `frequently()` here so the `0` case has more of a chance of being tested (since its more likely to cause issues than a random positive number). Also not sure why we only went up to 50 before, seems to me like we should be testing much larger ranges (probably worth going above `Integer.Max_VALUE`) so we make sure that we are ok for very high values since we expect aggregations to collect a lot of documents